ext/curl/interface.c

```diff
@@ -1520,6 +1520,11 @@ static bool php_curl_set_callable_handler(zend_fcall_info_cache *const handler_f
 		zend_fcc_dtor(handler_fcc);
 	}
 
+	if (Z_TYPE_P(callable) == IS_NULL) {
+		handler_fcc->function_handler = NULL;
```

Comment:
This shouldn't be required, as `zend_fcc_dtor(handler_fcc);` will set the handler to null already, and if not it is already initialized as such.

Reply:
Ah, I missed the fact that `memcpy` zeroes all fields.